---
title: "Identification Of Dysregulated Transcription Factor Activity In Temporal Lobe Epilepsy"
collection: publications
category: manuscripts
permalink: /publication/2015-10-01-paper-title-number-3
excerpt: 'Approximately a third of patients with epilepsy continue to have uncontrolled seizures despite appropriate treatment with available antiseizure medications (i.e. drug-resistant epilepsy). There is emerging evidence that transcription factor (TF) activity is both dysregulated in epilepsy and modifiable by small-molecule drugs, providing an opportunity for treatment innovation. Methods for identifying dysregulated TFs and their target genes are still in their nascent stage and the reproducibility of findings remains unclear. We aimed to determine the concordance of findings, in terms of the TFs dysregulated, across different studies of drug-resistant epilepsy and to evaluate the performance of different methods for identifying dysregulated TFs. We used publicly available single-nucleotide RNA-seq data to construct discovery and validation datasets comprising individuals with drug-resistant temporal lobe epilepsy and healthy controls. We found good concordance (83%, 105/126) between the pySCENIC (Python implemented Single-cell Regulatory Network Inference and Clustering) and hdWGCNA (high-dimensional Weighted Gene Co-expression Network Analysis) methods. Cell-type specific concordance across the discovery and validation datasets was low (36% 137/377) and this could be attributed, in part, to differences in data quality. In contrast, we found strong concordance between TFs that met strict concordance criteria in the current study with those implicated in a tissue-level study in patients with drug-resistant epilepsies, with the overlap being higher for TLE-related modules relative to modules for other drug-resistant epilepsies [86% (32/37) vs. 21% (18/84), Fisher’s exact test: 95% 7.40 to 85.5, p < 0.0001]. Most TFs identified had been reported as being associated with epilepsy in the overall literature (91%, 53/58). Our findings strengthen the hypothesis that TFs are key to the pathophysiology of drug-resistant epilepsy and could represent novel drug targets. We recommend that multiple methods be applied to optimise discovery.'
date: 2025-04-24
venue: 'medRxiv'
slidesurl: #'http://academicpages.github.io/files/slides3.pdf'
paperurl: 'https://www.medrxiv.org/content/10.1101/2025.04.24.25326321v1.full.pdf'
citation: 'Robert Zeibich, Terence J. O’Brien, Piero Perucca, Patrick Kwan, Alison Anderson. Identification of dysregulated transcription factor activity in temporal lobe epilepsy. <i>Preprint: medRxiv.</i> 2025 Apr 24. <a href="https://doi.org/10.1101/2025.04.24.25326321." target="_blank">View article online</a>.'
---

Approximately a third of patients with epilepsy continue to have uncontrolled seizures despite appropriate treatment with available antiseizure medications (i.e. drug-resistant epilepsy). There is emerging evidence that transcription factor (TF) activity is both dysregulated in epilepsy and modifiable by small-molecule drugs, providing an opportunity for treatment innovation. Methods for identifying dysregulated TFs and their target genes are still in their nascent stage and the reproducibility of findings remains unclear. We aimed to determine the concordance of findings, in terms of the TFs dysregulated, across different studies of drug-resistant epilepsy and to evaluate the performance of different methods for identifying dysregulated TFs. We used publicly available single-nucleotide RNA-seq data to construct discovery and validation datasets comprising individuals with drug-resistant temporal lobe epilepsy and healthy controls. We found good concordance (83%, 105/126) between the pySCENIC (Python implemented Single-cell Regulatory Network Inference and Clustering) and hdWGCNA (high-dimensional Weighted Gene Co-expression Network Analysis) methods. Cell-type specific concordance across the discovery and validation datasets was low (36% 137/377) and this could be attributed, in part, to differences in data quality. In contrast, we found strong concordance between TFs that met strict concordance criteria in the current study with those implicated in a tissue-level study in patients with drug-resistant epilepsies, with the overlap being higher for TLE-related modules relative to modules for other drug-resistant epilepsies [86% (32/37) vs. 21% (18/84), Fisher’s exact test: 95% 7.40 to 85.5, p < 0.0001]. Most TFs identified had been reported as being associated with epilepsy in the overall literature (91%, 53/58). Our findings strengthen the hypothesis that TFs are key to the pathophysiology of drug-resistant epilepsy and could represent novel drug targets. We recommend that multiple methods be applied to optimise discovery.