---
title: "Optimising Whole-Genome Sequencing Diagnostics In Epilepsy"
collection: publications
category: inprogress
permalink: /publication/2024-02-17-paper-title-number-4
excerpt: 'This paper is about fixing template issue #693.'
date: 2025-08-21
venue: 'undefined'
#paperurl: ''
#citation: ''
---

<p><strong>Background.</strong>  Whole-genome sequencing (WGS) is increasingly applied in clinical medicine, but 
best practices for variant detection, curation and interpretation have yet to be determined. We 
investigated how the choice of software methods and tools can impact WGS diagnostic yield.</p>  
<p><strong>Methods.</strong> WGS data from 70 patients enrolled in a randomised controlled trial had been 
processed using a commonly applied ‘gold standard’ workflow. Variants were curated and 
classified in accordance with the American College of Medical Genetics (ACMG) guidelines by 
a multidisciplinary team (MDT) comprising bioinformaticians, neurologists, and genetic 
counsellors. The same data were then processed using a range of different variant callers and 
annotation tools to evaluate their performance, and the diagnostic yield, in terms of pathogenic 
and likely pathogenic variants identified, was compared to conventional analysis.</p> 
<p><strong>Results.</strong> Alternative methods identified an additional 19 putative (likely) pathogenic single
nucleotide and InDel (short insertion/deletion) variants. While the choice of SNV/indel caller 
made no difference, the choice of quality cutoff or software version used for variant calling (n 
= 6), annotation tool used (n = 6), and new information becoming available (n = 7) impacted 
the diagnostic yield. For copy number variant detection, using multiple callers and a standard 
quality filter across them improved curation. Most new putative variants were heterozygous in 
genes associated with autosomal disease or found not to be phenotypically relevant to the 
patient. Three led to the diagnosis in unsolved trial patients, increasing the diagnostic yield 
from 23% (16/70) to 27% (19/70), and a further two have been referred for clinical validation.</p>  
<p><strong>Conclusion.</strong> This study highlights the importance of re-testing patients with no findings when 
new methods, technological developments and information become available.</p> 